packages/api/__tests__/index.js

```diff
@@ -277,31 +278,56 @@ describe('/api/profile/createUser', () => {
       done();
     });
   });
-
-  it('should send a 201 response when a student is created', (done) => {
-    request({
-      url,
-      method: 'POST',
-      body: testUser,
-      json: true,
-      resolveWithFullResponse: true,
-    }).then((res) => {
-      expect(res.statusCode).toBe(201);
-      expect(res.body).toMatchObject({ user_created: 'Success', user_email: testUser.email });
-      done();
+  it('should return 201 on creation of new user', (done) => {
+    tokenService.generate({
```

Comment:
You can make this function in `beforeAll`, so that no need to write in all test.